tools/config-markdown.py

```
@@ -1,207 +1,152 @@
-
-        anchor_links = generate_anchor_links(item)
-        anchor_links_content = "\n".join(anchor_links) + "\n\n"
+        item_dir = DOCS_DIR / item['id']
+        item_dir.mkdir(exist_ok=True)
 
-        file_name_technical = f"{item['id']}.technical.md"
-        file_path_technical = os.path.join(item_dir, file_name_technical)
-        markdown_content_technical = create_markdown_technical(item)
-        full_content_technical = anchor_links_content + markdown_content_technical
+        anchors = "\n".join(generate_anchor_links(item)) + "\n\n"
+        technical_md = create_markdown_technical(item)
```

Ruff (0.8.2)

Undefined name `create_markdown_technical`

(F821)